### PR TITLE
Create Text_Someone_Message_With_iPhone

### PR DESCRIPTION
**Pull request by brunoooost**
Including a payload that send a message to someone with iphone

### CODE REVIEW

**Comment by aleff-github** [comment]
Before accepting your payload, I would like to suggest a few improvements:

1.	Please add a README file that provides a technical explanation of what your payload does. You can refer to the README of [NexusWannaBe](https://github.com/aleff-github/my-flipper-shits/blob/main/Windows/Execution/Shutdown_After_1_Minute-By_NexusWannaBe/README.md) for guidance.

2.	Add a brief delay at the start of the program. When the `GUI r` command is executed, the operating system needs some time to process the device. To ensure proper execution, it’s recommended to introduce a short delay (even 1 or 2 seconds but however it depends) before proceeding with the actual execution.

3.	When executing a `STRING` command, it must complete 100% before proceeding. Therefore, adding a delay between the `STRING` command and the `ENTER` key might be unnecessary, unless you’ve encountered race conditions during testing where the device seems to prematurely consider the input complete. If that’s the case, it would be interesting to discuss and understand why this happens.

I haven’t had a chance to test your payload yet, but once I do, I’ll be happy to add it to the payload list.

Looking forward to your updates,
Aleff

**Comment by brunoooost** [comment]
Okey, i’ll do it. Enviado desde mi iPhoneEl 22 oct 2024, a las 11:34, Aleff ***@***.***> escribió:﻿
Before accepting your payload, I would like to suggest a few improvements:


Please add a README file that provides a technical explanation of what your payload does. You can refer to the README of NexusWannaBe for guidance.


Add a brief delay at the start of the program. When the GUI r command is executed, the operating system needs some time to process the device. To ensure proper execution, it’s recommended to introduce a short delay (even 1 or 2 seconds but however it depends) before proceeding with the actual execution.


When executing a STRING command, it must complete 100% before proceeding. Therefore, adding a delay between the STRING command and the ENTER key might be unnecessary, unless you’ve encountered race conditions during testing where the device seems to prematurely consider the input complete. If that’s the case, it would be interesting to discuss and understand why this happens.


I haven’t had a chance to test your payload yet, but once I do, I’ll be happy to add it to the payload list.
Looking forward to your updates,
Aleff

**Comment by brunoooost** [comment]
Hey, let me know 🙌

**Comment by aleff-github** [comment]
> Hey, let me know 🙌

Sorry for the delay, i'll check it asap, don't worry ^_^

**Comment by aleff-github** [comment]
> Okey so here is the payload with the 2 seg and the review of the enter and string: [Send_Message_ios.txt](https://github.com/brunoooost/Send_Messages_iOS/blob/main/send_message_ios) also the readme here: [README.md](https://github.com/brunoooost/Send_Messages_iOS/blob/main/README.md)

both links are broken, however you can upload yourself the readme file into the branch

**Review by brunoooost — COMMENTED** [review]
# Send_Message_iOS by bst04

## Description

This script, created by bst04, is designed to work on iOS systems. It opens the Messages app, navigates to a specific contact, and sends a predefined text message.

---

## Technical Explanation

1. **Initialization**:
    - `DELAY 2000`: Waits for 2 seconds to ensure the system is ready.
    - `GUI h`: Simulates pressing "Command" + "h" to hide the current application.
    - `DELAY 150`: Waits for 0.15 seconds.
    - `GUI SPACE`: Simulates pressing "Command" + "Space" to open Spotlight Search.
    - `DELAY 150`: Waits for 0.15 seconds for Spotlight to open.
    - `BACKSPACE`: Clears any existing text in the search bar.

2. **Open Messages App**:
    - `DELAY 500`: Waits for 0.5 seconds.
    - `STRING Messages`: Types "Messages" to search for the Messages app.
    - `DELAY 250`: Waits for 0.25 seconds.
    - `ENTER`: Opens the Messages app.

3. **Navigate to Contact**:
    - `DELAY 1500`: Waits for 1.5 seconds to ensure the Messages app is fully opened.
    - `GUI n`: Simulates pressing "Command" + "n" to start a new message.
    - `DELAY 250`: Waits for 0.25 seconds.
    - `STRING [persona]`: Types the name of the contact (replace `[persona]` with the actual name).
    - `DELAY 500`: Waits for 0.5 seconds.
    - `ENTER`: Selects the contact.
    - `DELAY 250`: Waits for 0.25 seconds.
    - `TAB`: Navigates to the message input field.
    - `ENTER`: Confirms the selection.
    - `DOWNARROW`: Moves to the message input field.

4. **Send Text Message**:
    - `DELAY 500`: Waits for 0.5 seconds.
    - `STRING [texto]`: Types the predefined message (replace `[texto]` with the actual text).
    - `DELAY 500`: Waits for 0.5 seconds.
    - `ENTER`: Sends the message.

---

## Usage

1. Copy the payload script to your DuckyScript-enabled device.
2. Ensure the device is connected to an iOS system.
3. Execute the payload to open the Messages app, navigate to a specific contact, and send the predefined text message.

File: iOS/Prank/Text_Someone_Message_With_iPhone [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+DELAY 1500
+GUI n
+DELAY 250
+STRING [person]
+DELAY 500
+ENTER
+DELAY 250
+TAB
+ENTER
+DOWN ARROW
```
Comment (aleff-github) [inline iOS/Prank/Text_Someone_Message_With_iPhone:25]: This looks like an error

**Pull request merged**